src/templates/_partials/_page_header.twig

```diff
@@ -1,15 +1,15 @@
 {% if title is defined and title %}
-
-<div>
-
-  <h1>{{ title }}</h1>
-
-  {% if intro is defined and intro %}
-    <div>
-      {{ intro }}
-    </div>
-  {% endif %}
-
-</div>
-
+  <div class="redactor flex flex-col gap-4 py-[30px] lg:py-[32px]">
+    {% include "_partials/_heading.twig" with {
+      text: title,
+      htag: 'h1'
+    } %}
+
+    {% if intro is defined and intro %}
+      <h4 class="lg:w-2/3 font-light text-gray-700">
+        {{ intro }}
+      </h4>
```

Comment:
Use `p` tag instead of `h4`

Remember that heading tags (h1 to h6) are use to section the content and should only be used as such.